<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoEagleEyeDialog</class>
 <widget class="QDialog" name="InfoEagleEyeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonCloseInfo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Schlie√üen</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/image/image/hsLogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>10</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelProject">
      <property name="text">
       <string>Project</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelNames">
      <property name="text">
       <string>Namen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelVersion">
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
